Specs/Scene/Vector3DTileClampedPolylinesSpec.js

```diff
@@ -147,7 +147,8 @@ describe(
         );
 
         var features = [];
-        tileset.createFeatures(mockTileset, features);
+        var content = tileset._root._content;
+        content._polylines.createFeatures(mockTileset, features);
```

Comment:
Is this the right idea to access `Vector3DTileClampedPolylines.createFeatures()`? Web debugger shows all the `content` fields are undefined.